.github/workflows/check-links.yml

```diff
@@ -1,4 +1,4 @@
-name: Check Documentation Links
+name: "Check documentation links"
 
 # documentation is tested only on:
```

Comment:
@dkastl I had the files ready so that when transifex bot makes the PR only the Spanish translation links are checked.
Transifex bot does not change the English links, it does not change code, And I think that you started the changes from develop and trying to merge to master.

Why don't you make another PR departing from master but:
Only rename files and do the lint, but don't change the logic of the files.
All has to be done in one commit so that the changes can be ported to develop and to release-3.0

Reply:
Sorry, the `master` branch was not intentional. Fixed that.
Will work on the other issues later.